staging/src/k8s.io/apiserver/pkg/server/options/priority_and_fairness.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"fmt"
+
+	"github.com/spf13/pflag"
+	utilflowcontrol "k8s.io/apiserver/pkg/util/flowcontrol"
+	"k8s.io/utils/path"
+)
+
+// PriorityAndFairnessOptions holds the api server priority and fairness options.
+type PriorityAndFairnessOptions struct {
```

Comment:
Maybe FlowControlOptions?

@MikeSpreitzer - WDYT?

Reply:
"flow control" is a broader term, and is even used elsewhere in Kubernetes to mean other things.  I think we should stick with the more specific term.

Reply:
If anything, I would rather change the existing places where "flow control" refers to APF to use the more specific term.  Sadly, one of them is in the API Group, so that will be a heavy lift.  But I am willing to lend a hand!

Reply:
For now renamed everything to "flow control". I don't think it would hurt to have the ability to specify any parameter related to this particular package (above priority and fairness).

Reply:
Let's stick to the discussion in https://github.com/kubernetes/kubernetes/pull/111222#discussion_r926966132